### PR TITLE
Map SuperModel to ServiceModel

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

**Comment by hakonhall** [comment]
Adds ServiceModel - a thin wrapper around a map of ApplicationInstances: A ServiceModel differs from SuperModel in that it uses the application model classes, contains service status, and also has artificial applications (config server application).

Adds SlobrokMonitor2, which is a thin wrapper around SlobrokList and Mirror, and used when generating the ServiceModel. Will be renamed SlobrokMonitor once that can be removed.

**Comment by hakonhall** [comment]
The next steps is to set up listeners of SuperModel changes, and expose ServiceModel (application model classes) through the ServiceMonitor interface, which is the interface implemented by the SlobrokAndConfigIntersector.

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/SlobrokMonitor2.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            case "config-sentinel":
+            case "configproxy":
+            case "configserver":
+            case "filedistributorservice":
+            case "logd":
+            case "logserver":
+            case "metricsproxy":
+            case "slobrok":
+            case "transactionlogserver":
+            case "ytracecleaner":
```
Comment (hmusum) [inline service-monitor/src/main/java/com/yahoo/vespa/service/monitor/SlobrokMonitor2.java]: ytracecleaner is not in use anymore

**Review by hmusum — APPROVED** [review]
Just one nitpick, merge at will

File: service-monitor/src/main/java/com/yahoo/vespa/service/monitor/SlobrokMonitor2.java (hunk 1 above)
Comment (hakonhall) [reply service-monitor/src/main/java/com/yahoo/vespa/service/monitor/SlobrokMonitor2.java]: Fixed

**Comment by hakonhall** [comment]
Travis Java build succeeded.

**Pull request merged**